vignettes/customizing-dccvalidator.Rmd

```diff
@@ -14,8 +14,24 @@ knitr::opts_chunk$set(
 )
 ```
 
-dccvalidator is intended to be customizable for different settings. The Shiny
-application uses a [configuration file](https://github.com/Sage-Bionetworks/dccvalidator/blob/master/config.yml)
+dccvalidator is intended customizable for different settings, however the
+built-in Shiny application is designed to ensure that data conforms to a
+specific project organization structure in which data is organized into studies
+that are described with a combination of metadata files. These metadata files
+document the individuals, specimens, and assay(s) that were performed in the
```

Comment:
It could be useful to add some commentary describing the design thinking for this metadata structure - to join across files on individualID and specimenID variables. Might be useful to note those variables are defined in the `complete_columns` configuration option.

Reply:
Good idea! I've added more context, but I didn't want to mention the config file until the "Customizing dccvalidator" section. So I have instead added more information to the documentation of the `complete_columns` option further down in the document.